assets/collage.css

```diff
@@ -42,6 +38,10 @@
   border: 0.1rem solid rgba(var(--color-foreground), 0.08);
 }
 
+.collage-card.collage-collection {
+  border: none;
+}
```

Comment:
Ah I saw the comment on the issue but still seems odd that all other blocks get it except this one.

Reply:
I had a conversation with @wiktoriaswiecicka and @Oliviammarcello and we decided it made sense because of the border that would get added next to the photo. This is also the only block with a possible image and text and all color schemes.